frappe/__init__.py

```diff
@@ -1469,12 +1469,12 @@ def _load_app_hooks(app_name: str | None = None):
 	for app in apps:
 		try:
 			app_hooks = get_module(f"{app}.hooks")
-		except ImportError:
+		except ImportError as e:
 			if local.flags.in_install_app:
 				# if app is not installed while restoring
 				# ignore it
 				pass
-			print(f'Could not find app "{app}"')
+			print(e)
 			if not request:
 				raise SystemExit
```

Comment:
This `SystemExit` needs to be fixed too. It's silently returning 0 exit code :sweat:

Reply:
can't edit PR, doing it separately.